src/test/resources/regressions/features/multisets/multiset-range-fail1.gobra

```diff
@@ -5,5 +5,5 @@ package pkg
 
 func foo() {
   //:: ExpectedOutput(type_error)
-  assert mset[1 .. 4] == set[1 .. 4]
```

Comment:
I think this is a case where the original test file had the spaces already(git blame says since 2 years). I accidentally removed them because I just ran a RegEx over the whole folder to remove the spaces I added, but I can try to restore test files that already had the spaces even before I added them to all cases.